src/gtk/font.cpp

```diff
@@ -373,6 +379,13 @@ wxFontWeight wxFont::GetWeight() const
     return M_FONTDATA->m_nativeFontInfo.GetWeight();
 }
 
+int wxFont::GetNumericWeight() const
+{
+    wxCHECK_MSG( IsOk(), wxFONTWEIGHT_MAX, wxT("invalid font") );
```

Comment:
`wxS()` is slightly more efficient (no conversion needed), so if you care about this, it can be used, but I don't think it's really a big concern for the assert messages (we definitely don't care about speed of asserting, so it's just a few extra bytes in the library size) and using nothing is simpler and more clear.

Reply:
thanks for the clarification